components/logstash/scripts/configure.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+from os.path import (
+    dirname,
+    join,
+)
+
+from cloudify import ctx
+
+ctx.download_resource(
+    join('components', 'utils.py'),
+    join(dirname(__file__), 'utils.py'))
+import utils  # NOQA
+
+
+CONFIG_PATH = 'components/logstash/config'
+LOGSTASH_SERVICE_NAME = 'logstash'
+
+
+def configure_logstash():
+
+    logstash_conf_path = '/etc/logstash/conf.d'
+
+    rabbitmq_username = ctx.instance.runtime_properties['rabbitmq_username']
```

Comment:
for safety, perhaps use **ctx.instance.runtime_properties.get('rabbitmq_username')**